demosplan/DemosPlanCoreBundle/StoredQuery/AbstractStoredQuery.php

```diff
@@ -63,7 +63,7 @@ private function digestForIndex($string): string
              * which is exactly our last index :)
              *
              */
-            $result .= self::DIGEST_CHARACTERS[(ord($hash[$ii]) & 0x3F)];
```

Comment:
Not sure about this one as this could potentially be a logic change even if I don't think so?

Reply:
It's not a logic change. I'd assume sometime in the past, another function or operator was mixed into this and later removed, leaving behind the parenthesis.